packages/storage/src/database/common/setting.ts

```diff
@@ -20,13 +21,22 @@ export class ModelDbSetting extends ModelBasic<DbSetting> {
     return ModelDbSetting.instances.get(key)!;
   }
 
-  public async updateSetting(setting: DbSetting) {
+  public async updateSetting(setting: Partial<DbSetting>) {
```

Comment:
This can be done quite easier
```ts
public async updateSetting(setting: Partial<DbSetting>) {
  return this.collection.updateOne(
    {},
    { $set: Object.fromEntries(Object.entries({}).filter(([k]) => ['merkleHeight', 'appPublicKey'].includes(k))) },
    { upsert: true },
  );
}
```

Reply:
Done